free-1.md

```diff
@@ -5,47 +5,108 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+    The user is someone who has some familiarity with programming languages and 
+    would want a way to schedule their week quickly.
+
 ### What's the need?
 
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+    The need met by this idea is to quickly make schedules using text. This
+    could help lots of people by being able to quickly visualize schedules
+    and make changes to them. This person's experience right now is probably
+    making weekly schedules through google calendar or some other equivalent.
+    If this language could help them, that could mean generating weekly
+    schedules or course calendars could be automated and done quickly.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+    I think a DSL is appropriate because calendars sometimes feel too cluttered?
+    This might be a me problem, but when making an event on google calendar I
+    have 8 options for how it repeats, the option to add notes, google meet links,
+    a visibility option, location options, guest permissions, and I don't need all
+    of that I just need it to repeat weekly and have a start/end time and name.
+    It would be nice if there were a text-based DSL that could quickly generate
+    calendars by taking in some input, which could then be automated by Python or
+    some other external language. 
```

Comment:
If I was honest, I don't use calendars for this very reason. So in many ways I like your idea. I manly use a very full google document list of things to do. In my heart of hearts tho I know if this was a DSL it would be used by programmers only. So it isn't really a general purpose tool without some sort of languages processing.